library/Rules/Callback.php

```diff
@@ -13,26 +13,45 @@
 
 namespace Respect\Validation\Rules;
 
-use Respect\Validation\Exceptions\ComponentException;
+use function array_shift;
+use function array_unshift;
+use function call_user_func_array;
+use function func_get_args;
 
+/**
+ * This is a wildcard validator, it uses a function name, method or closure.
+ *
+ * @author Alexandre Gomes Gaigalas <alexandre@gaigalas.net>
+ * @author Henrique Moody <henriquemoody@gmail.com>
+ * @author William Espindola <oi@williamespindola.com.br>
+ */
 class Callback extends AbstractRule
```

Comment:
`final`

Reply:
@henriquemoody I don't put the final statement because `FilterVar` extends this class.
```
➜  Validation git:(issue-921-apply-contribution-guidelines-to-callback-rule) ./vendor/bin/phpunit

Fatal error: Class Respect\Validation\Rules\FilterVar may not inherit from final class (Respect\Validation\Rules\Callback) in /Users/williamespindola/Desktop/Validation/library/Rules/FilterVar.php on line 18
```

Reply:
Fair enough...